test/test_integration_policy.py

```diff
@@ -116,6 +116,31 @@ def test_policy_bad_syntax_raises_error(self):
         self.assertIn("422 Unprocessable Entity {\"error\":{\"code\":\"validation_failed\",\"message\":",
                   str(mock_log.output))
 
+    @integration_test()
+    def test_policy_bad_syntax_error_user_recieve_detailed_error(self):
+        policy_with_bad_syntax = "- ! user bad syntax"
+        with redirect_stderr(self.capture_stream):
+            output = utils.load_policy_from_string(self, policy_with_bad_syntax, exit_code=1)
+        self.assertIn("{\"error\":{\"code\":\"validation_failed\",\"message\":", output, 
+                        "Could not find detailed error in the output."
+                        "Expected to find:"
+                        "{\"error\":{\"code\":\"validation_failed\",\"message\":"
+                        "Result:"
+                        f"{output}")
+
+    @integration_test()
+    def test_policy_load_bad_branch_recieve_detailed_error(self):
+        with redirect_stderr(self.capture_stream):
+            output = self.invoke_cli(self.cli_auth_params,
+                           ['policy', 'load', '-b', 'wrong-branch', '-f', 
+                           self.environment.path_provider.get_policy_path("resource")], exit_code=1)
+        self.assertIn("{\"error\":{\"code\":\"not_found\",\"message\":", output, 
```

Comment:
does this tests works? looks like you need regex check. but I don't really know what should be the output

Reply:
yes, apparently it does work, as you can see the pipeline passed and it passes locally

Reply:
sorry I thought the error code should be a number